cmd/ebrelayer/txs/parser.go

```diff
@@ -61,14 +62,14 @@ func EthereumEventToEthBridgeClaim(valAddr sdk.ValAddress, event types.EthereumE
 	amount := sdk.NewIntFromBigInt(event.Value)
 
 	// Nonce type casting (*big.Int -> int)
-	nonce := int(event.Nonce.Int64())
+	nonce := int(4000000000)
```

Comment:
We should have a comment here explaining why we're using a large int for a nonce